README.md

```diff
@@ -146,6 +146,8 @@ The community maintained a list of **Algorithms and Data Structures** implementa
       <td><code>gr-dij</code></td>
       <td class="text-center"></td>
       <td class="text-center">&#10004;</td>
+      <td class="text-center">&#10004;</td>
+      <td class="text-center">&#10004;</td>
```

Comment:
Hi @riturajjain2000 
Why two languages are ticked in here?

Reply:
Oh ,sorry I added it by mistake, removed the extra line